includes/gating/functions.php

```diff
@@ -143,7 +143,7 @@ function maybe_restrict_content( string $content ) : string {
 	$content_excerpt = $post_obj->post_excerpt;
 	$public_content  = '';
 	$cta_button_html = sprintf(
-		'<p><a href="%1$s" class="coil-serverside-message-button">%2$s</a></p>',
+		'<p><a target=”_blank” href="%1$s" class="coil-serverside-message-button">%2$s</a></p>',
```

Comment:
Just check the type of " that you're using, maybe you copy and pasted these in?

Need to use the classic `"` not the fancy `”`

Reply:
I did paste it. I will fix it.